clerk/organizations.go

```diff
@@ -228,6 +229,11 @@ func (s *OrganizationsService) addMembersSearchParamsToRequest(r *http.Request,
 			query.Add("user_id", userID)
 		}
 	}
+	if params.Roles != nil {
```

Comment:
:upside_down_face: No need to check for `nil` here. If the slice is `nil` the range won't iterate at all. 

https://go.dev/play/p/-yM16GQvnOY